editor/plugins/script_text_editor.cpp

```diff
@@ -1724,6 +1724,39 @@ static String _quote_drop_data(const String &str) {
 	return escaped.quote(using_single_quotes ? "'" : "\"");
 }
 
+static String _get_dropped_resource_line(const Ref<Resource> &p_resource, bool p_create_field) {
+	const String &path = p_resource->get_path();
+	const bool is_script = p_resource->is_class_ptr(Script::get_class_ptr_static());
+
+	if (!p_create_field) {
+		return vformat("preload(%s)", _quote_drop_data(path));
+	}
+
+	StringName class_name = p_resource->get_class_name();
```

Comment:
```suggestion
	const StringName &class_name = p_resource->get_class_name();
```

Reply:
I actually overwrite this value a couple lines later

Reply:
The whole method was simplified as I don't need the class name anymore